### PR TITLE
fix: re-generate clients with some template overrides, move module up into repo root

### PR DESCRIPTION
**Pull request by shiqi-synctera**
PR does a couple things
- re-generates the client with latest public openapi spec, using some more helpful template overrides
- moves the whole library up to project root, in order for the library to be go-get-able
- update readme to give more modern import instructions
- this version should work correctly with oneOf / allOf / anyOf discriminators

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,379 +77,373 @@ All URIs are relative to *https://api.synctera.com/v0*
...
-*WebhooksApi* | [**GetWebhook1**](synctera/docs/WebhooksApi.md#getwebhook1) | **Get** /webhooks/{webhook_id} | Get a webhook
-*WebhooksApi* | [**ListEvents**](synctera/docs/WebhooksApi.md#listevents) | **Get** /webhooks/{webhook_id}/events | List webhook events
-*WebhooksApi* | [**ListWebhooks1**](synctera/docs/WebhooksApi.md#listwebhooks1) | **Get** /webhooks | List webhooks
-*WebhooksApi* | [**ReplaceSecret**](synctera/docs/WebhooksApi.md#replacesecret) | **Put** /webhooks/secret | Replace an existing secret
-*WebhooksApi* | [**ResendEvent**](synctera/docs/WebhooksApi.md#resendevent) | **Post** /webhooks/{webhook_id}/events/{event_id}/resend | Resend an event
-*WebhooksApi* | [**RevokeSecret**](synctera/docs/WebhooksApi.md#revokesecret) | **Delete** /webhooks/secret | Revoke the secret
-*WebhooksApi* | [**TriggerEvent**](synctera/docs/WebhooksApi.md#triggerevent) | **Post** /webhooks/trigger | Trigger an event
-*WebhooksApi* | [**UpdateWebhook**](synctera/docs/WebhooksApi.md#updatewebhook) | **Put** /webhooks/{webhook_id} | Update a webhook
+*AccountsApi* | [**CreateAccount**](docs/AccountsApi.md#createaccount) | **Post** /accounts | Create an account
```
Comment (kris-hansen) [inline README.md:80]: These paths won't route from the README

File: LICENSE [hunk 2]
```diff
@@ -1,201 +0,0 @@
-                                 Apache License
```
Comment (kris-hansen) [inline LICENSE]: This file is required

**Review by kris-hansen — CHANGES_REQUESTED** [review]

File: LICENSE (hunk 2 above)
Comment (shiqi-synctera) [reply LICENSE]: will add back in. That's one accidental deletion

File: README.md (hunk 1 above)
Comment (shiqi-synctera) [reply README.md:80]: do they not? Worked fine for me :thinking:

File: LICENSE (hunk 2 above)
Comment (shiqi-synctera) [reply LICENSE]: done

File: README.md (hunk 1 above)
Comment (kris-hansen) [reply README.md:80]: Sorry, I see now you eliminated the synctera/ path which was in the former archive

**Review by kris-hansen — APPROVED** [review]

**Pull request merged**